pkgs/development/libraries/hare-ev/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, fetchFromSourcehut
+, hare
+, unstableGitUpdater
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hare-ev";
+  version = "unstable-2022-12-29";
+
+  src = fetchFromSourcehut {
+    owner = "~sircmpwn";
+    repo = pname;
+    rev = "c585f01f4d13a25edb62477c07fdf32451417fee";
+    hash = "sha256-lB+ZPKGeYASV9oCE5iyDUCCPu2V07hqMXEktIY4fn1E=";
+  };
+
+  nativeCheckInputs = [
+    hare
+  ];
+
+  preConfigure = ''
+    export HARECACHE=$(mktemp -d)
+  '';
+
+  installFlags = [ "PREFIX=" "DESTDIR=$(out)" ];
```

Comment:
Is there a workaround in the context of #65718?

Reply:
updated the PR to use `installFlags = [ "PREFIX=$(out)" ];`, which seems to be the preferred approach?

for reference the project uses a [custom Makefile](https://git.sr.ht/~stacyharper/bonsai/tree/f19084d23246f14c04c69c0f7ceda48d419a4063/item/Makefile#L29) -- no autotools. `PREFIX` and `DESTDIR` are used only in the Makefile, and don't appear anywhere in the code so probably not relevant to the program's runtime:
```Makefile
install:
        mkdir -p $(DESTDIR)$(PREFIX)/bin
        install -m755 bonsaid $(DESTDIR)$(PREFIX)/bin
        install -m755 bonsaictl $(DESTDIR)$(PREFIX)/bin
```

Reply:
oh, one sec i only updated `bonsai` -- let me look into `hare-ev` and `hare-json` too.

Reply:
okay, `hare-ev` and `hare-json` are safe with `PREFIX=$(out)` too 👍️ thanks for pointing me to those better practices.